scripts/ci/stats/merged_prs.py

```diff
@@ -92,11 +92,12 @@ def process_pr(pr):
     business_days = sum(1 for day in dates if day.weekday() < 5)
     prj['business_days_open'] = business_days
 
+    release_notes = 'Release Notes' in labels and len(labels) == 1
```

Comment:
This could hit some false positives though, right? Lke you could have release notes and a gigantic change at a path that has no areas, how unlikely is that?

Reply:
Wondering if it wouldn't be better to change the set assignee script to actually check the list of changed files and tag as trivial instead

Reply:
yeah, can be done this way as well, we will need to add the Trivial label then, this was supposed to work without the Trivial label.